Notebooks/OpenIssues.github-issues

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "# Project Mu GitHub Open Issue Dashboard\r\n\r\nThis notebook displays [Project Mu](https://microsoft.github.io/mu/) open issues."
+  },
+  {
+    "kind": 2,
+    "language": "github-issues",
+    "value": "// list of project mu repos\r\n$repos=repo:repo:microsoft/mu repo:microsoft/mu_basecore repo:microsoft/mu_plus repo:microsoft/mu_tiano_plus repo:microsoft/mu_oem_sample repo:microsoft/mu_tiano_platforms repo:microsoft/mu_siicon_arm_tiano repo:microsoft/mu_silicon_intel_tiano repo:microsoft/mu_common_intel_min_platform repo:microsoft/mu_devops repo:microsoft/mu_feature_config repo:microsoft/mu_feature_mm_supv repo:microsoft/mu_feature_ipmi repo:microsoft/mu_feature_uefi_variable repo:microsoft/mu_crypto_release repo:microsoft/mu_pip_environment repo:microsoft/mu_pip_python_library repo:microsoft/mu_pip_build repo:microsoft/mu_build repo:microsoft/mu_common_intel_adv_features"
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "📬 All Open Issues"
+  },
+  {
+    "kind": 2,
+    "language": "github-issues",
+    "value": "$repos is:open is:issue archived:false"
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "All Open Issues with no labels"
+  },
+  {
+    "kind": 2,
+    "language": "github-issues",
+    "value": "$repos is:open is:issue archived:false no:label"
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "All Open Issues with no assignee"
+  },
+  {
+    "kind": 2,
+    "language": "github-issues",
+    "value": "$repos is:open is:issue archived:false no:assignee"

```

Comment:
Could this one also track `state:needs-owner`?

So, no assignee or the needs owner label is present.

Reply:
I can make the query do that, but in testing, it doesn't give us what we want (a unified list of any issues that has either label:state:needs-owner and no:asignee)

Because label:state:needs-owner and no:asignee are separate query identifiers, it would create a query that only includes items that contain both.

If we could ensure that new issues get "label:state:needs-owner" automatically applied, we could just rely on that.

In the meantime, would having two separate queries satisfy our needs? Then we can remove the no:asignee query at a later date?

Reply:
Separate queries is fine.